src/main/java/kr/codesquad/Application.java

```diff
@@ -0,0 +1,15 @@
+package kr.codesquad;
+
+import java.util.Scanner;
+
+public class Application {
+    private Command command = new Command(new Scanner(System.in));
+
+    public void start() {
+        int manCount = command.commandManCont();
```

Comment:
command.commandManCont(); 에서 단어가 겹치고 오타가 있네요.
메소드 이름은 동사로 시작하는 것이 좋습니다.

```suggestion
        int manCount = command.countMan();
```